lms/static/sass/elements/_system-feedback.scss

```diff
@@ -111,10 +111,24 @@
     }
   }
 
+  &.recovery-email-alert {
+    background: $state-info-bg !important;
```

Comment:
I think we should avoid `!important` from these values unless they are necessary.

Reply:
we are adding this class in recovery email case only so need it to override the existing one.

Reply:
I just Removed.